src/yang/ietf-tcp.yang

```diff
@@ -82,7 +82,8 @@ module ietf-tcp {
       type boolean;
       default "false";
       description
-        "Enable support of TCP-Authentication Option (TCP-AO).";
+        "Enable support of TCP-Authentication Option (TCP-AO) when set
+         to true.";
```

Comment:
Would it not be better to say:

"When set to true, TCP-Authentication Option (TCP-AO) is enabled."

Reply:
looks good to me.

Reply:
Fixed